src/rateEngine/LoadProfile.ts

```diff
@@ -156,7 +158,7 @@ class LoadProfile {
 
   aggregate(otherLoadProfile: LoadProfile): LoadProfile {
     return new LoadProfile(
-      this._loadProfile.map((loadHour, idx) => {
+      (this._loadProfile ?? []).map((loadHour, idx) => {
```

Comment:
why is the default necessary here?

Reply:
Got to the same conclusion when removing the business logic from this.expanded().